public/scripts/tags.js

```diff
@@ -746,8 +750,16 @@ async function handleTagImport(character, { forceShow = false } = {}) {
     const folderTags = getOpenBogusFolders();
 
     // Choose the setting for this dialog. If from settings, verify the setting really exists, otherwise take "ASK".
-    const setting = forceShow ? tag_import_setting.ASK
-        : Object.values(tag_import_setting).find(setting => setting === power_user.tag_import_setting) ?? tag_import_setting.ASK;
+    let setting;
+    if (forceShow) {
+        setting = tag_import_setting.ASK;
+    } else if (importAll) {
+        setting = tag_import_setting.ALL;
+    } else if (importExisting) {
+        setting = tag_import_setting.ONLY_EXISTING;
+    } else {
+        setting = Object.values(tag_import_setting).find(setting => setting === power_user.tag_import_setting) ?? tag_import_setting.ASK;
+    }
```

Comment:
That's a bit repetitive. How about we change it and don't supply three different boolean switches, but an actual `manualImportSettingsOverride` or whatever, that is default null. But can also be all of the other options.

That also covers the `forceShow` case by simply supplying `tag_import_setting.ASK` there.

Slims down the optional args for both those functions, and is what you actually wanna do anyway.